### PR TITLE
normalize the log trace data

### PR DESCRIPTION
**Pull request by Jim3Things**
This change normalizes the log trace structures into redux.

The organizer class is removed.

I've also pulled the 'max severity' icon on the spans, as that has a lot of computational load and I'm not sure how much help it gives.

The complex redux slices are split into their own modules.

Initial unit tests for the normalizeLog* functions added.

### CODE REVIEW

**Comment by Jim3Things** [comment]
Committing and retaining for review, due to travel.

File: clients/observer/src/store/NormalizeLog.test.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        linkSpanID: "",
+        linkTraceID: "",
+        impacted: [],
+    })
+
+    const traceEntry3: Entry = new Entry({
+        name: "testSpan3",
+        spanID: "CH001",
+        parentID: "0002",
+        traceID: "000000004",
```
Comment (MikeCJohnsonUk) [inline clients/observer/src/store/NormalizeLog.test.tsx:97]: Is this supposed to have the same traceId value as traceEntry2?

**Review by MikeCJohnsonUk — COMMENTED** [review]
Just a couple of probably minor questions...

File: clients/observer/src/store/NormalizeLog.test.tsx (hunk 1 above)
Comment (Jim3Things) [reply clients/observer/src/store/NormalizeLog.test.tsx:97]: Yes.  The traceID is the tag that groups multiple spans together into a 'logical unit'.  We don't do much with it, but that seems to be the case.

File: clients/observer/src/store/NormalizeLog.tsx [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+// arrays contain the keys to remove.
+export type NormalizedLogRemoval = {
+    removeEntries: string[],
+    removeEvents: string[],
+    removeLinks: string[],
+    removeRoots: string[],
+}
+
+function linkFQName(sourceSpan: string, sourceTrace: string, linkId: string): string {
+    if (linkId === "" || linkId === undefined) {
```
[the remark this answers is not in the document]
Comment (Jim3Things) [reply clients/observer/src/store/NormalizeLog.tsx:62]: If the test were to a field in linkId, then yes.  I believe that either order should be equivalent when there are no sub-fields involved.

File: clients/observer/src/store/NormalizeLog.test.tsx (hunk 1 above)
Comment (MikeCJohnsonUk) [reply clients/observer/src/store/NormalizeLog.test.tsx:97]: Ok

File: clients/observer/src/store/NormalizeLog.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (MikeCJohnsonUk) [reply clients/observer/src/store/NormalizeLog.tsx:62]: Ok.

**Comment by MikeCJohnsonUk** [comment]
Ok.

**Pull request merged**